gdx/src/com/badlogic/gdx/scenes/scene2d/actions/ParallelAction.java

```diff
@@ -68,7 +68,9 @@ public boolean act (float delta) {
 		try {
 			Array<Action> actions = this.actions;
 			for (int i = 0, n = actions.size; i < n && actor != null; i++) {
-				if (!actions.get(i).act(delta)) complete = false;
+				if (currentAction.getActor()!=null && !currentAction.act(delta))

```

Comment:
Yes, I have broke merge somehow.
I made this merge again, now should be fine. Sorry for that :)